qiskit/tools/visualization/_utils.py

```diff
@@ -48,6 +48,135 @@ def _trim(image):
     return image
 
 
+def _get_layered_instructions(circuit, reversebits=False, justify=None):
+    """
+    Given a circuit, return a tuple (qregs, cregs, ops) where
+    qregs and cregs are the quantum and classical registers
+    in order (based on reversebits) and ops is a list
+    of DAG nodes which type is "operation".
+    Args:
+        circuit (QuantumCircuit): From where the information is extracted.
+        reversebits (bool): If true the order of the bits in the registers is
+            reversed.
+        justify (str) : `left`, `right` or `none`. Defaults to `left`. Says how
+            the circuit should be justified.
+    Returns:
+        Tuple(list,list,list): To be consumed by the visualizer directly.
+    """
+
+    if justify:
+        justify.lower()
+    # default to left
+    justify = justify if justify in ('right', 'none') else 'left'
```

Comment:
Should `None` be `'none'` and the default be `'none'`?

Reply:
I set the default to be `'left'` as this was the initial suggestion in [this comment](https://github.com/Qiskit/qiskit-terra/issues/1685#issuecomment-455627702) and it was also said to be a [common format](https://github.com/Qiskit/qiskit-terra/issues/1707#issuecomment-457405877). I think there are merits to having `'left'` and to having `'none'` as the default, but I would be inclined to stick with `'left'` if this is the format users would be used to seeing.